src/aspire/classification/rir_class2d.py

```diff
@@ -334,7 +339,9 @@ def _sk_nn_classification(self, coeff_b, coeff_b_r):
 
     def _legacy_nn_classification(self, coeff_b, coeff_b_r):
         """
-        Perform nearest neighbor classification.
+        Perform nearest neighbor classification using port of ASPIRE legacy MATLAB code.
+
+        Note `distances` are dot products, ie "corr".
```

Comment:
regular quotes instead of `code quotes`?

Reply:
Mentioned that its a returned var.